src/DataGrid.tsx

```diff
@@ -68,58 +54,94 @@ export interface DataGridProps<R, K extends keyof R> {
    * 4. Update all cells under a given cell by double clicking the cell's fill handle.
    */
   onGridRowsUpdated?<E extends GridRowsUpdatedEvent<R>>(event: E): void;
```

Comment:
May be `onRowsUpdated` would make more sense. Wdyt?

Reply:
For this one, I would say maybe keep it as it is, since the event is actually `GridRowsUpdatedEvent`, so the names are consistent.

Reply:
I think we can also rename `GridRowsUpdatedEvent` to RowsUpdateEvent`